src/Api/getCityWeather.tsx

```diff
@@ -0,0 +1,18 @@
+const baseUrl = "http://api.weatherapi.com";
+const endpointPath = "v1/current.json";
+
+const getUrl = (city: string) => {
+	const searchParams = new URLSearchParams({
+		key: "c6068278a47d4030aad124640241101",
+		q: city,
+		aqi: "no",
+	});
+	return `${baseUrl}/${endpointPath}?${searchParams.toString()}`;
+};
+
+const fetchDataCountry = async (url) => {
```

Comment:
By reading this function name I'd guessed that kind of data I'd get. `fetchCityWeather` will be more descriptive